### DISCUSSION TITLE
Notification toggle is missing

### DISCUSSION THREAD

**Discussion by sudo-bmitch** [community/community#8216] [Mobile] [answered]
The notification toggle seems to be missing from the app. This is on Android, version 1.41.0.

![Screenshot_20211129-074513.png](https://user-images.githubusercontent.com/1979858/143872953-5463e084-a1c8-4f04-92f3-4028ff2a99dc.png)

**Comment by eliperkins** [comment, marked as answer]
Thanks for the feedback @sudo-bmitch ! The ability to subscribe or unsubscribe has moved to the `•••` (overflow) menu within the header of the issue. Tapping that will display a menu with the option to subscribe or unsubscribe:

<img width="431" alt="Screen Shot 2021-12-02 at 3 12 26 PM" src="https://user-images.githubusercontent.com/1051453/144678345-1a99c197-acac-469d-bfb5-581ecee0b2a0.png">

**Reply by sudo-bmitch** [reply]
Thanks, I'm seeing it for issues and some pull requests, but the menu isn't available on all PRs yet. 🤔

![Screenshot_20211203-174934.png](https://user-images.githubusercontent.com/1979858/144682975-28b97a15-523a-487c-ae52-c28ffd41b2c4.png)

**Reply by sudo-bmitch** [reply]
Looks like this is now working on all the issues, PRs, and discussions I checked. Thanks!
